### ISSUE TITLE
ICICI bank home page! radio button  is not announce their value

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#14]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2.press H for heading till to reach to ‘I mobile -pay ’heading level 5and two times tab to reach tab ‘right product for you ‘
3. Reached to the tab ‘find the right product for you ‘
4.press tab  reached to combo box  and select ‘insurance’
5. press tab to reach next combo box and select ‘securing my loved one’
6.press tab and then reached to the radio button
expected Result the radio button should announced their  value
Actual Result the screen reader announced only select radio button three of five,  four of five and five of five‘
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value

**Comment by manish10** [comment]
The grammar in the title needs to be correct.
In this case, it seems the label for the radio button is not announced with the radio buttons. It is ok but it will be great if you can provide the specific label in the expected result so that developers know exactly what needs to be fixed. Is the label missing only on buttons 3,4, and 5 and being spoken correctly on buttons 1 and 2?
